### DISCUSSION TITLE
WriteOutPut_GiD.for(55): #error: unknown fpp directive.

### DISCUSSION THREAD

**Discussion by GeoCalcul** [Anura3D/Anura3D_OpenSource#167] [Q&A] [answered]
Hello All,
I have this error: " WriteOutPut_GiD.for(55): #error: unknown fpp directive."
line 55 is : #pragma comment(lib, "gidpost.lib").
They used # for comment?
should I change it to !.
regrads

**Comment by murphy-john** [comment, marked as answer]
# is a C pre-compiler action. 

Add a "!" to the beginning of the line.
